### PR TITLE
Add rollback button for fast, accessible rollbacks to the last successful release

### PR DESCRIPTION
**Pull request by drshrey**
screen shot 2018-07-25 at 11 57 28 pm

### CODE REVIEW

File: src/components/Project/Secrets/index.js [hunk 1]
```diff
@@ -382,6 +382,7 @@ export default class Secrets extends React.Component {
         <Loading />
       )
     }
+    console.log(project)
```
Comment (aballman) [inline src/components/Project/Secrets/index.js]: Ixnay the onsolecay

File: src/components/Project/Releases/index.js [hunk 2]
```diff
@@ -588,7 +597,7 @@ export default class Releases extends React.Component {
   }
 
   renderDrawer(){
-		if (this.state.drawerRelease === null){
+		if (this.state.drawerRelease == null){
```
Comment (aballman) [inline src/components/Project/Releases/index.js]: Why the change here?

Comment (drshrey) [reply src/components/Project/Releases/index.js]: mistake, i was creating 2 state props so drawerRelease was being ignored. fixed now

File: src/components/Project/Releases/index.js [hunk 3]
```diff
@@ -588,6 +594,7 @@ export default class Releases extends React.Component {
   }
 
   renderDrawer(){
+    console.log(this.state.drawerRelease)
```
Comment (aballman) [inline src/components/Project/Releases/index.js]: Please remove
